data/config-schema/src/main/resources/net/corda/schema/configuration/db/1.0/corda.db.json

```diff
@@ -64,6 +64,30 @@
                   "type": "null"
                 }
               ]
+            },
+            "idleTimeout": {
+              "description": "The maximum time (in minutes) a connection can stay idle in the pool. A value of 0 means that idle connections are never removed from the pool.",
```

Comment:
@simon-johnson-r3 I believe we don't want to keep the "_A value of 0 means that idle connections are never removed from the pool_" option in which case we should go for `"minimum": 1`?

Reply:
An interesting question, I'm not an expert on db connection tuning although I doubt we'd ever use it. That said I don't see keeping 0 as an option a problem myself.